docs/pullsubscription/README.md

```diff
@@ -68,44 +72,47 @@ Pub/Sub topic, such as
    gcloud pubsub topics create $TOPIC_NAME
    ```
 
-1. Update `TOPIC_NAME` in the [`pullsubscription.yaml`](pullsubscription.yaml)
-   and apply it.
+1. Replace the placeholders with real values.
 
-   If you're in the samples directory, you can replace `TOPIC_NAME` and apply in
-   one command:
+      1. Placeholders:
```

Comment:
I don't understand why this is now a substep, there are only a few steps, seems to make sense to just have steps 1-4.